test cases/common/270 int_to_str_fill/meson.build

```diff
@@ -0,0 +1,6 @@
+project('test_fill_in_int_to_string')
+
+assert(4.to_string() == '4')
+assert(4.to_string(fill: 3) == '004')
+assert(4.to_string(fill: -3) == '4')
+assert(-4.to_string(fill: 3) == '-04')
```

Comment:
I think CI failure reveals a parser issue, it calls to_tring on 4 then tried minus on the resulting string. Probably just put parenthesis around -4.

Reply:
> I think CI failure reveals a parser issue, it calls to_tring on 4 then tried minus on the resulting string. Probably just put parenthesis around -4.

Just saw this after pushing. I put the integer into a variable instead, is that ok?

Reply:
That works too